### PR TITLE
Add completion for composer

### PR DESCRIPTION
**Pull request by tonglil**

### CODE REVIEW

File: composer-completion.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class ComposerCompletion < Formula
+  homepage "https://gist.github.com/tonglil/753d59d6d649f85600fe"
+  url "https://gist.githubusercontent.com/tonglil/753d59d6d649f85600fe/raw/854fcb2f5a4cae824119d6c5c40a2bde4e541689/composer-completion.sh"
+  sha1 "21bedfccf0a9cdf23b7b1a7fd5fee4188a72fadd"
```
Comment (bfontaine) [inline composer-completion.rb]: Please use a `sha256` here.

**Comment by bfontaine** [comment]
Ping?

**Comment by bfontaine** [comment]
Closing due to no response; feel free to re-submit a PR that passes `brew audit --strict`!

**Comment by tonglil** [comment]
@bfontaine Sorry this lapsed my response.

I'm not sure what I would need for the `test do` section, and the versioning of the completion is on gist. Can you review it and advise?

https://github.com/Homebrew/homebrew-completions/compare/master...tonglil:composer-completion

**Comment by bfontaine** [comment]
@tonglil You need to add a `version "..."` line that correspond to the version. You could use e.g. `version "1"` to start, like [`rake-completion.rb`](https://github.com/Homebrew/homebrew-completions/blob/9fb2be4a5c2711a005a114ae3aaea66504193201/rake-completion.rb#L5).

Regarding the test something like this should work:
```rb
test do
    assert_match "-F _composer",
      shell_output("source #{bash_completion}/composer-completion.sh && complete -p composer")
end
```

**Comment by tonglil** [comment]
@bfontaine thanks, I will update this this week.

**Pull request closed without merging**